### PR TITLE
change the scale thresh generally

### PR DESCRIPTION
**Pull request by WeiweiZhang1**
change the scale thresh generally
0.0 --->  1e-5
The effect of scale clip was evaluated on 4 commonly used models with acceptable accuracy fluctuations.
Adding scale clip prevents weight values from disappearing and reduces model instability after quantization.

### CODE REVIEW

File: auto_round/data_type/int.py [hunk 1]
```diff
@@ -59,8 +59,6 @@ def quant_tensor_asym(weight, bits=4, v=0, min_scale=1.0, max_scale=1.0, scale_d
     wmax[tmp] = +1
     scale = ((wmax - wmin) / maxq).to(scale_dtype)
     scale = torch.clamp(scale, min=q_scale_thresh)
-    if (scale == 0.).any():
-        scale = torch.clamp(scale, min=1e-5)
```
Comment (wenhuach21) [inline auto_round/data_type/int.py:63]: where is the clip? better using q_scale_thresh

**Review by wenhuach21 — APPROVED** [review]

File: auto_round/quantizer.py [hunk 2]
```diff
@@ -46,7 +46,7 @@ def reshape_tensor(v, group_size=-1):
 
 def quant_tensor(
         quant_func, data, bits=4, group_size=-1, v=0, min_scale=1.0, max_scale=1.0, scale_dtype=torch.float16,
-        weight_min=None, weight_max=None, q_scale_thresh=0.0, **kwargs,
+        weight_min=None, weight_max=None, q_scale_thresh=1e-5, **kwargs,
```
Comment (wenhuach21) [inline auto_round/quantizer.py:49]: we have a thresh value in wrapper linear, better pass that value

**Comment by wenhuach21** [comment]
add clip in unwrapper function

**Review by wenhuach21 — APPROVED** [review]
